components/TextFields/src/MDCTextInputController.h

```diff
@@ -64,7 +64,14 @@
 
  Default is red.
  */
-@property(nonatomic, nullable, strong) UIColor *errorColor UI_APPEARANCE_SELECTOR;
+@property(nonatomic, nullable, strong) UIColor *errorColor;
+
+/**
+ Default value for above property.
+
+ Class property. Replaces UIAppearance for classes that do not inherit from UIView.
+ */
+@property(class, nonatomic, null_resettable, strong) UIColor *errorColorDefault;
```

Comment:
"Default value for `errorColor`.

Default is red."

Omit discussion of implementation (class property) or side discussions (UIAppearance).